src/tools/kdb/CMakeLists.txt

```diff
@@ -21,7 +21,15 @@ if (BUILD_SHARED)
 	add_executable (kdb $<TARGET_OBJECTS:kdb-objects>)
 	add_dependencies (kdb kdb_gen_templates_generated)
 
-	target_link_libraries (kdb elektra-core elektra-kdb elektratools elektra-opts elektra-merge)
+	find_package (Threads)
+	target_link_libraries (
+		kdb
+		${CMAKE_THREAD_LIBS_INIT}
```

Comment:
Why does the kdb tool need to be linked with pthreads? I thought it would be enough to only link `jni` with pthreads.

Maybe @markus2330 knows as it was his suggestion?

Reply:
See #3884

Reply:
It is not really explained there either.

Reply:
That's true, only mentioned it to give some reference, to where it comes from.

Reply:
Technically it is needed if any library requires it. As the libraries are depending on which plugins are in use we are in a gray area. I think I suggested it because of some segfault we had in JNI which looked like that the missing pthread could be the cause. Iirc it was actually not the case. But linking against it also does not harm.

Reply:
When I'm remembering correctly, I think it was part of the problem.